usr/share/rear/rescue/default/020_create_skeleton_dirs.sh

```diff
@@ -0,0 +1,34 @@
+# Recreate some directories/symlinks in the rescue system if present.
+# Eventually more stuff from $SHARE_DIR/skel could be moved here.
+# The main use is for symlinks, as their presence in the source tree
+# (under $SHARE_DIR/skel) brings some problems.
+# See https://docs.fedoraproject.org/en-US/packaging-guidelines/Directory_Replacement/
+
+# Other file types can be copied using this mechanism,
+# but it is probably better to use the COPY_AS_IS variable for them.
+# For a directory the difference is that COPY_AS_IS would copy it recursively,
+# while this mechanism recreates the directory as empty.
+
+local skeleton_paths=( /var/run )
+
+local d
+local actual_skeleton_paths=()
+
+# some checks before copying
+for d in "${skeleton_paths[@]}" ; do
```

Comment:
Please avoid one-letter variable names in general
even if it is currently rather obvious here that
'd' means 'directory' but when this script may get enhanced
over time that 'd' likely is kept as is forever even when
later several directory-like thingies are needed and then
we would get a mess of meaningless variables names
like the original 'd' plus 'dir' and 'd2' and whatnot, cf.
https://github.com/rear/rear/wiki/Coding-Style#code-must-be-easy-to-read

Reply:
done in last commit [3678f6c](https://github.com/rear/rear/pull/3380/commits/3678f6c321565dbd1d5475a453c20a7f28cc8355)